android/build.gradle

```diff
@@ -25,8 +25,8 @@ rootProject.allprojects {
 apply plugin: 'com.android.library'
 
 def MIN_IAP_SDK_VERSION = '1.5.6'
-def COMPILE_SDK_VERSION = 28
-def TARGET_SDK_VERSION = 28
+def COMPILE_SDK_VERSION = 30
```

Comment:
Do we have to bump up the target SDK version? Is that required by Flutter?

Reply:
No but we are upgrading for all so i have upgrade this also.